www/js/model.js

```diff
@@ -45,6 +49,17 @@ define([
 					};
 				}
 			},
+			setGraphHelpShown: function(_isGraphHelpShown){
+                this.model.properties._isGraphHelpShown = _isGraphHelpShown;
```

Comment:
doesnt state have this property already? what is the need to save it to the model?

Reply:
This helps us not to make call for state's variables everytime they are needed